sydorovaos/first/task3.py

```diff
@@ -0,0 +1,7 @@
+"""Знайти суму цілих чисел із діапазону дійсних чисел"""
+first = float(input())
+last = float(input())
+sum = 0
+for i in range(int(first+1),int(last+1)):
```

Comment:
You shouldn't call the value in `for` like this.
That would be better:
`for numbers in range(int(first+1),int(last+1)):`